packages/module/package.json

```diff
@@ -31,7 +31,9 @@
     "branches": "main"
   },
```

Comment:
IIRC the `release` key needs to be removed from the package file for semantic-release to look for a release config file.